ansible/roles/test/tasks/single_lag_test.yml

```diff
@@ -31,9 +31,9 @@
 - name: test fanout interface (physical) flap and lacp keep correct po status follow minimum links requirement
   include: lag_minlink.yml
   vars:
-    wait_down_time: 20
+    wait_down_time: 35
```

Comment:
Why is the wait time here much less than in L#48 ? Where do these numbers come from?

Reply:
I don't know who decided on these values or how they were chosen. I increased these a bit to make the test succeed more reliably on certain platforms. I believe they could still be increased more, but I believe someone needs to investigate this further and choose the most appropriate values.